src/components/FloatingDropdown/FloatingDropdown.st.css

```diff
@@ -40,3 +75,33 @@
 .optionDivider {
     -st-mixin: divider;
 }
+
+.option {
+    -st-extends: DropdownOption;
+}
+
+.root .dropdown::dropdownContent {
+    background-color: value(DropdownBackgroundColor);
+}
+
+.root .dropdown::dropdownContent .optionDivider {
+    background-color: value(ItemTextColor);
+}
+
+.root .dropdown::dropdownContent::dropdownOption:hovered,
+.root .dropdown::dropdownContent::dropdownOption:selected {
+    background-color: applyOpacity(value(ItemTextColor), 0.06);
+}
+
+.root .dropdown::dropdownContent .option::title {
+    color: value(ItemTextColor);
+    font: value(TextFont);
+}
+
+.floatingDropdownBase {
+    -st-extends: FloatingDropdownBase;
+    -st-mixin: overrideStyleParams(
+        MainTextColor value(MainButtonTextColor),
+        MainTextFont value(TextFont)
+    );
+}
```

Comment:
Maybe we should add the overrideStyleParams class to this component as well. 
Since we are now defining the API for the users we could only expose the `overrideStyleParams API`. It really improves the performance.

Reply:
changed